Insurgency/insserver

```diff
@@ -71,19 +114,19 @@ consolelogdate="${consolelogdir}/${servicename}-console-$(date '+%d-%m-%Y-%H-%M-
 # Do not edit
 
 fn_runfunction(){
+functiondir="${rootdir}/functions"
```

Comment:
I will do this later tonight. I actually have a slightly more involved version that abstracts the downloader a bit more, allowing fetching of not only functions but any files, and I have the rudimentary versioning system for the function files. I'll make one branch for just the functiondir fix to keep it small, and then later you can take a look at the other downloader I have and see if it looks good to you.